code/modules/integrated_electronics/core/assemblies.dm

```diff
@@ -577,3 +577,20 @@
 	name = "type-b electronic drone"
 	icon_state = "setup_drone_arms"
 	desc = "It's a case, for building mobile electronics with. This one is armed and dangerous."
+
+/obj/item/device/electronic_assembly/attack_animal(mob/living/simple_animal/user)
```

Comment:
This looks like a copypaste of the /obj proc, if you use the revised attack_animal on /item you might be able to skip this

Reply:
it's copypaste from mech.IDK, why it doesn't work with item override alone.